deepchem/feat/basic.py

```diff
@@ -23,18 +23,26 @@ def _featurize(self, mol):
     mol : RDKit Mol
         Molecule.
     """
-    from rdkit.Chem import Descriptors
+    try:
+      from rdkit.Chem import Descriptors
+    except ModuleNotFoundError:
+      raise ValueError("This class requires RDKit to be installed.")
     wt = Descriptors.ExactMolWt(mol)
     wt = [wt]
     return wt
```

Comment:
Many featurizer return np.array. So `return wt` -> `return np.asarray(wt)` is better

Reply:
Done!